.devcontainer/Dockerfile

```diff
@@ -1,20 +1,17 @@
 # For details, see https://github.com/devcontainers/images/tree/main/src/ruby
 FROM mcr.microsoft.com/devcontainers/ruby:1-3.3-bookworm
 
-# Install Rails
-# RUN gem install rails webdrivers
-
+# Update existing node version, keep in sync with .nvmrc
```

Comment:
Is there any way to dynamically read this from .nvmrc at this stage in build process?

Reply:
If you run `nvm install`, it should install the version from `.nvmrc` if it exists. You can add a `COPY` step before this command to copy this specific file.